docs/content/components/state.md

```diff
@@ -43,7 +43,7 @@ Component for rendering the status of an item.
 | Name | Type | Default | Description |
 | :- | :- | :- | :- |
 | `title` | `String` | N/A | `title` HTML attribute. |
-| `color` | `Symbol` | `:default` | Background color. One of `:default`, `:green`, `:red`, or `:purple`. |
+| `color` | `Symbol` | `:default` | Background color. One of `:open`, `:closed`, `:merged`, `:default`, `:green`, `:red`, or `:purple`. |
```

Comment:
It's a little odd to have this argument named `color`, as that's also a system argument. But perhaps that's a concern for another time 😄

Reply:
I think this is part of the conversation about splitting `system_arguments` into different things